lib/sepa.js

```diff
@@ -287,9 +287,6 @@
     /** Id assigned to the creditor */
     creditorId: '',
 
-    /** Original creditor id (optional, for creditor id migrations) */
-    originalCreditorId: null,
```

Comment:
Looks like we should bump the major version if we're removing this. I guess we could provide backwards compatibility, though might not be worth the hassle. What do you think?

Reply:
SGTM, let's do it.